config/default.yml

```diff
@@ -51,6 +51,12 @@ Minitest/AssertNil:
   Enabled: true
   VersionAdded: '0.1'
 
+Minitest/AssertPathExists:
+  Description: 'This cop enforces the test to use `assert_path_exists` instead of using `assert(File.exist?(path))`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-path-exists'
```

Comment:
It's generally better to link to the website instead - https://minitest.rubystyle.guide/ The guide looks much nicer there.